<launch>
  <node name="voice_aiui" pkg="robot_audio" type="voice_aiui_node" output="screen"/>
</launch>
